### ISSUE TITLE
The training problem of dataset 20X

### ISSUE THREAD

**Issue by kiven1213** [vqdang/hover_net#138]
Hi，@simongraham and  @vqdang
First of all, I would like to sincerely thank you for your contribution, and I have successfully run it on the Pannuke data set, and got a good result. But now I have a problem, I hope you can help me, I will be very grateful. I want to subsample the Pannuke data to 20X, and then use this code for model training to predict other 20X data sets. Where should I modify the code? Is it convenient to point it out?

**Comment by vqdang** [comment]
@kiven1213, you can either resize the patches beforehand by modifying the `extract_patches` or you can do it on the flight in augmentation process here. 
https://github.com/vqdang/hover_net/blob/a0f80c7acb9a14964d597d15d7ebd9688a0230cb/dataloader/train_loader.py#L111
Also, make sure that you are using Nearest interpolation when resizing the mask, else there will be artifactsin ground truth.

**Comment by kiven1213** [comment]
hi,@vqdang
Thank you very much for your patience. That is to say, this model is also suitable for 20x data set, right? I just need to sample the training data set to 20x, and I don't need to modify the code. Is that right?

**Comment by simongraham** [comment]
@kiven1213 you will also need to modify some of the post processing code because this is specifically tailored to work at 40x. For example, you will need to modify the arguments in functions such as the one [here](https://github.com/vqdang/hover_net/blob/master/models/hovernet/post_proc.py#L46) and the one[ here](https://github.com/vqdang/hover_net/blob/master/models/hovernet/post_proc.py#L56). There will be a few more, but basically you should appropriately reduce the size of the kernels used and look out for some additional functions such as `remove_small_objects()`.

**Comment by simongraham** [comment]
Of course also make sure to also change [this command line argument](https://github.com/vqdang/hover_net/blob/master/run_infer.py#L66) if processing WSIs. This should answer your questions, so will now close the issue.

**Comment by Tato14** [comment]
@kiven1213, have same issue here. I would like to know if you could share the Pannuke 20X dataset. Also, have any experience with `remove_small_objects` `min_size` and `cv2.Sobel` `ksize` arguments when using Pannuke at 20X?
Thanks!

**Comment by Tato14** [comment]
@simongraham 

> you should appropriately reduce the size of the kernels used

you mean reducing `ksize` in [Hovernet](https://github.com/vqdang/hover_net/blob/61ffeca6d22bba8aae0727523192150175be8b18/models/hovernet/net_desc.py#L14) model? All `Conv2d`?
